frontend/src/pages/Home.jsx

```diff
@@ -31,34 +52,355 @@ class Home extends Component {
           </Typography>
           <Button 
             variant="contained" 
-            color="primary"
             style = {{
-              marginRight: 10
+              marginRight: 10,
+              color: "#0F1111",
+              background: "#f0c14b",
+              fontWeight: 600
+            }}
+            onClick = {()=>{
+              this.props.setLogin_dialog(true)
+              this.props.setRegister_dialog(false)
             }}
           >
-            Sign in 
-          </Button>
-          <Button variant="contained" color="primary">
-            Register
+            Log In
           </Button>
+          
+          {/* This is for the login dialog */}
+          <Dialog
+            open={this.props.loginDialogOpen} 
+            onClose={()=>{
+              this.props.setLogin_dialog(false)
+              this.props.setRegister_dialog(false)
+            }}
+            aria-labelledby="login-dialog-title"
+            style = {{
+              margin: "auto",
+              width: 500
+            }}
+          >
+
+            <DialogTitle id="login-dialog-title"> 
+              Login
+              <IconButton
+                style = {{
+                  position: "absolute",
+                  top: "5px",
+                  right: "5px"
+                }}
+                onClick={()=>{
+                  this.props.setLogin_dialog(false)
+                  this.props.setRegister_dialog(false)
+                }}
```

Comment:
maybe can be replaced by one of the `handleClickXXX` handler below(?

Reply:
Good advice, I'll add a method to avoid duplication